src/pocketmine/entity/Living.php

```diff
@@ -384,13 +384,12 @@ public function attack(EntityDamageEvent $source){
 			$e = $source->getDamager();
 			if($source instanceof EntityDamageByChildEntityEvent){
 				$e = $source->getChild();
-			}
-
-			if($e !== null){
```

Comment:
Why has this condition been removed?

Reply:
🤦‍♂️ woops, I thought only `EntityDamageByEntityEvent::getDamager()` calls `Server::findEntity()`.